### ISSUE TITLE
Problem importing LAS files with VLR header mismatch.

### ISSUE THREAD

**Issue by wintal** [PDAL/PDAL#4298]
One vendor (Trimble) is writing las files with an error in the las file header - they write that there are 2 VLRs, but there's only actually 1. This gets reported as a warning by lastools, ignored by cloud compare and PDAL rejects it. This is arguably correct behaviour (file could be corrupt), but it would be good if there were a way to ignore this error and continue. 

We've reported the issue back to Trimble, but given these las files are coming directly out of their scanners it may not get fixed in practice for a while.

I've attached both an artificial example (utm17.las, modified to reproduce the problem) and the original (badheader.las) which I've had to zero the actual data in (but left the header intact), since I don't have permission to share it. 

[lasfiles.zip](https://github.com/PDAL/PDAL/files/13916946/lasfiles.zip)

```
$ pdal translate utm17.las pleaseletmereadit.las
```

**Expected behavior**
The ability to configure the las reader to ignore this class of error. 

**System/installation information:**
I'm running from a built version of pdal - I've tested 2.3 and master.

**Comment by wintal** [comment]
I've pushed a PoC of what a fix could look like here: https://github.com/PDAL/PDAL/pull/4299

**Comment by abellgithub** [comment]
Others can chime in, but I'm not a fan. If your data is bad, fix it. Here's a program you can run:

```
#!python3
import sys

if len(sys.argv) != 2:
    sys.exit("Usage: fixtrimble <filename>")

filename = sys.argv[1]
with open(filename, "rb+") as f:
    f.seek(100)
    vlrs = int.from_bytes(f.read(4), byteorder='little')
    print("Number of VLRS = {0}. Fix? ".format(vlrs))
    response = input()
    if response != 'Y' and response != 'y':
        exit();
    f.seek(100)
    f.write((vlrs - 1).to_bytes(4, byteorder='little'))
```

**Comment by hobu** [comment]
> One vendor (Trimble) is writing las files with an error in the las file header 

Which Trimble software and versions is writing bunk data?

**Comment by wintal** [comment]
@hobu The Trimble software causing issues is Trimble Access. I'm not sure if it's only failing on certain settings (I don't have the software to test with) but we have a couple of customers that are hitting this issue. 

@abellgithub It's certainly possible to pre-process the file to fix it (there's various software they can round trip it through which fixes it). Our users are surveyors rather than programmers, so they're unlikely to be happy running python code to fix something but there are other ways to solve it. I'm not sure where people using pdal directly sit. Currently they would have to write code (or use other software) to process the files. 

I can certainly work around the problem without changes in PDAL, but I figured since all the other libraries/software I've tried read the file without issues I'd report it.
